src/test/e2e/helpers/database-init.ts

```diff
@@ -110,15 +120,20 @@ export default async function init(
     const testDb = createDb(config);
     const stores = await createStores(config, testDb);
     stores.eventStore.setMaxListeners(0);
+    const defaultRolePermissions = await getDefaultEnvRolePermissions(testDb);
     await resetDatabase(testDb);
     await setupDatabase(stores);
+    await restoreRolePermissions(testDb, defaultRolePermissions);
```

Comment:
re-apply the permissions

Reply:
One alternative solution would be to store all 15 permissions in a file. What are the tradeoffs?
* with DB query and insert approach our tests are verifying actual permissions in migrations
* one drawback of DB query and insert is that it's different from all the other data that we seed from database.json